Automattic/jetpack
# Shortcodes: minify jQuery Cycle script

I noticed this on my new site elio.blog and searched for an issue here and there's this one that this PR now fixes #3335. I know that #5620 exists but I'm not sure if we need to minify these kind of files on each build since we won't be changing them often. So, here's this PR that specifically solves #3335.

Also, the reason why I noticed this script was because after running a test in the site I saw the script loaded, and I wasn't using it at all. I realized that when Infinite Scroll is supported, this script and others are always loaded. However, in my tests this wasn't true. I tested with this same branch and whenever the post that had a slideshow gallery was loaded, the script was loaded, so it's not necessary to always load it.

#### Changes proposed in this Pull Request:
- use minified jQuery Cycle JS
- don't load the script always when Infinite Scroll is activated

#### Testing instructions:
* add a slideshow gallery and verify it works normally
* test Infinite Scroll and add a slideshow in a post that is displayed after IS fetches more entries and verify it works


#### Proposed changelog entry for your changes:
Shortcodes: the jQuery Cycle script used by slideshow galleries is now minified resulting in faster loading times.

cc @lancewillett for review
